src/vs/workbench/contrib/remote/browser/remote.ts

```diff
@@ -254,7 +239,8 @@ class HelpModel {
 				viewModel.helpInformation.map(info => (new HelpItemValue(commandService,
 					info.extensionDescription,
 					(typeof info.remoteName === 'string') ? [info.remoteName] : info.remoteName,
-					info.virtualWorkspace
+					info.virtualWorkspace,
+					info.reportIssue
```

Comment:
I don't think this will be enough to make sure that the `reportIssue` gets used instead of the Issue Reporter. You'll also need to change the `IssueReporterItem` `takeAction`:

https://github.com/microsoft/vscode/blob/b2e999b112435ccd8a46536fdd723026240f298a/src/vs/workbench/contrib/remote/browser/remote.ts#L422-L424

Reply:
With the raced timeout you can still get unlucky and have `action.description` be `''`. We could bump the timeout but TBH I should look into why the Codespaces extension seems to take so long to activate in the first place...